festim/exports/derived_quantities/average_volume.py

```diff
@@ -3,9 +3,40 @@
 
 
 class AverageVolume(VolumeQuantity):
+    """
+    Computes the average value of a field in a given volume
+    int(f ds) / int (1 * ds)
```

Comment:
Volume integration?

Reply:
Yeah, good catch! I thought I had gone through and fixed all these